src/adaPets.js

```diff
@@ -5,33 +5,54 @@ const result = require('./result.js');
 const setResult = result.setResult;
 const setError = result.setError;
 
-const BASE_URL = 'http://localhost:3000/pets/';
-
 // Option functions.
 const listPets = () => {
-  // Fill out as part of Wave 1.
+  axios.get('http://localhost:3000/pets')
+    .then((response) => {
+      setResult(response.data);
+    })
+    .catch((error) => {
+      setError('made up error message for wave one list pets');
+  });
 };
 
 const showDetails = (selectedPetId) => {
   if (!selectedPetId) {
-    setError("You tried to show details for a pet without selecting it!");
-    
-  }
-
-  // Fill out as part of Wave 2.
+    setError("You tried to show details for a pet without selecting it!"); 
+  } else {
+    let url = `http://localhost:3000/pets/${selectedPetId}`
```

Comment:
Does this need to be `let`? If you aren't going to change the value of `url` during this scope, you might as well use `const`.